src/Utilities/DoubleDicts.jl

```diff
@@ -3,116 +3,108 @@ module DoubleDicts
 import MathOptInterface
 
 const MOI = MathOptInterface
-const MOIU = MOI.Utilities
-const CI = MOI.ConstraintIndex
-const AD = AbstractDict
 
-const DataTypePair = Tuple{DataType,DataType}
+abstract type AbstractDoubleDict{V} <: AbstractDict{MOI.ConstraintIndex,V} end
```

Comment:
If we end up with this simple implementation...
We should rename this to ConstraintIndexedDict and reserve DoubleDict for future flexible implementation. Or ConstraintIndexDict or something along those lines.

DoubleDictInner could be SingleTypeConstraintIndexedDict.

IndexDoubleDict would become ConstraintIndexMap...